jeremy2918/tsseract
# Delete Requests

# [Delete request](https://trello.com/c/kghHzBhB/44-i-want-to-create-all-the-missing-delete-controllers-for-users-post-comments-etc)

## Description
This branch creates the missing routes and controllers to delete the different documents in the database.


## Testing Steps

- `git pull`
- `git checkout feat/delete-requests`
- `npm install`
- `npm run tsc`
- `npm run server`
- You can now check that there are existing routes to delete every document in the database.



## Changes
No UI Changes
